crates/turborepo-lib/src/package_graph/builder.rs

```diff
@@ -63,6 +67,7 @@ impl<'a> PackageGraphBuilder<'a> {
             package_manager: None,
             package_jsons: None,
             lockfile: None,
+            quiet: false,
```

Comment:
Mostly for testing because we don't want the logging outputs to end up in our integration tests, since we're running the Rust hashing code silently before executing the Go code.